Services/AzineMailgunTwigExtension.php

```diff
@@ -1,10 +1,24 @@
 <?php
 namespace Azine\MailgunWebhooksBundle\Services;
 
+use Azine\MailgunWebhooksBundle\DependencyInjection\AzineMailgunWebhooksExtension;
+use Azine\MailgunWebhooksBundle\Entity\Repositories\MailgunEventRepository;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Class AzineMailgunTwigExtension
+ * Provides some filters and global variables
+ * @package Azine\MailgunWebhooksBundle\Services
+ */
 class AzineMailgunTwigExtension extends \Twig_Extension
 {
-    public function __construct()
+    private $repository;
+    private $emailDomain;
+    
+    public function __construct(MailgunEventRepository $repository, $emailDomain)
```

Comment:
undo all changes to this file and get the required information in the controller.